benchmarking/simulation/mock_algorithm.py

```diff
@@ -35,13 +35,20 @@
 class MockAlgorithm:
     @staticmethod
     def get_team_generation_options(
-        num_students: int, num_teams: int, initial_teams: List[TeamShell] = None
+        num_students: int,
+        num_teams: int,
+        initial_teams: List[TeamShell] = None,
+        min_team_size: int = None,
+        max_team_size: int = None,
     ) -> TeamGenerationOptions:
         _num_teams = len(initial_teams) if initial_teams else num_teams
-        min_team_size = num_students // _num_teams
-        max_team_size = (
-            min_team_size if num_students % _num_teams == 0 else min_team_size + 1
-        )
+        if min_team_size and not max_team_size or max_team_size and not min_team_size:
```

Comment:
I know AND is done first over OR, but maybe some brackets here since it's not immediately clear imo